sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2504,26 +2482,16 @@ object GpuOverrides {
     expr[ArrayMin](
       "Returns the minimum value in the array",
       ExprChecks.unaryProject(
-        TypeSig.commonCudfTypes + TypeSig.DECIMAL_64 + TypeSig.NULL,
+        // TODO add back in STRING support once https://github.com/rapidsai/cudf/issues/9156
+        //   is fixed
+        TypeSig.commonCudfTypes + TypeSig.DECIMAL_64 + TypeSig.NULL - TypeSig.STRING,
         TypeSig.orderable,
-        TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL_64 + TypeSig.NULL),
+        TypeSig.ARRAY.nested(TypeSig.commonCudfTypes + TypeSig.DECIMAL_64 + TypeSig.NULL -
+            TypeSig.STRING),
         TypeSig.ARRAY.nested(TypeSig.orderable)),
       (in, conf, p, r) => new UnaryExprMeta[ArrayMin](in, conf, p, r) {
         override def tagExprForGpu(): Unit = {
-          if (isOrContainsFloatingPoint(in.dataType)) {
-            if (!conf.isFloatAggEnabled) {
-              willNotWorkOnGpu("the GPU will aggregate floating point values in" +
-                  " parallel and the result is not always identical each time. This can cause" +
-                  " some Spark queries to produce an incorrect answer if the value is computed" +
-                  " more than once as part of the same query.  To enable this anyways set" +
-                  s" ${RapidsConf.ENABLE_FLOAT_AGG} to true.")
-            }
-            if (conf.hasNans) {
-              willNotWorkOnGpu("Min aggregation on floating point columns that can contain NaNs" +
-                  " will compute incorrect results. If it is known that there are no NaNs, set" +
-                  s" ${RapidsConf.HAS_NANS} to false.")
-            }
-          }
+          checkAndTagFloatNanAgg("Min", in.dataType, conf, this)
```

Comment:
Looks like there's a missing call to `checkAndTagFloatAgg` here?

Reply:
No that was a bug in the old code. min and max always produce the correct answer (assuming no NaNs) because the order of operations does not matter for them in floating point.